src/common/lua/ngx_wasm_lua_ffi.c

```diff
@@ -368,8 +368,8 @@ ngx_int_t
 ngx_wa_ffi_shm_kv_get(ngx_wa_shm_t *shm, ngx_str_t *k, ngx_str_t **v,
     uint32_t *cas)
 {
-    ngx_int_t   rc;
-    unsigned    unlock = 0;
+    ngx_int_t  rc;
+    unsigned   unlock = 0;
```

Comment:
Note the spacing is correct here: 2 empty spaces + an extra space for potential pointers.